modules/cnv-viewing-virtual-machine-logs-web.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-logs-events.adoc
+
+[id="cnv-viewing-virtual-machine-logs-web_{context}"]
+= Viewing virtual machine logs in the web console
+
+Get virtual machine logs from the associated virtual machine launcher pod.
+
+.Procedure
+
+. In the {ProductName} console, click *Workloads* -> *Virtual Machines*.
+. Click the virtual machine to open the *Virtual Machine Details* screen.
+. In the *Overview* tab, click the `virt-launcher-<name>` pod in the *POD* 
```

Comment:
Pod

Reply:
Updated all instances of this, and Build and Deployment (for these files)